website/content/docs/connect/transparent-proxy.mdx

```diff
@@ -9,6 +9,35 @@ description: |-
 
 # Transparent Proxy
 
+Transparent proxy allows applications to communicate through the mesh without changing their configuration.
+It also hardens application security by preventing direct inbound connections that bypass the mesh.
+
+#### Without Transparent Proxy
+
+![Diagram demonstrating that without transparent proxy, applications must "opt in" to connecting to its dependencies through the mesh](/img/consul-connect/without-transparent-proxy.png)
+
+Without transparent proxy, application owners need to:
+
+1. Explicitly configure upstream services, choosing a local port to access them.
+1. Change application to access `localhost:<chosen port>`.
+1. Configure application to listen only on the loopback interface to prevent unauthorized
+   traffic from bypassing the mesh.
+
+#### With Transparent Proxy
+
+![Diagram demonstrating that with transparent proxy, connections are automatically routed through the mesh](/img/consul-connect/with-transparent-proxy.png)
+
+With transparent proxy:
+
+1. Upstreams are inferred from service intentions, so no explicit configuration
+   is needed.
+1. Outbound connections to KubeDNS names "just work" — rules redirect them
+   through the proxy.
```

Comment:
```suggestion
1. Outbound connections pointing to a KubeDNS names resolve and connect. Internal rules redirect them through the proxy.
```
I'm trying to make this statement seem less magical. The rules part could probably use a better description.